person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :is_of_age?
```

Comment:
- [ ] the method is called `of_age?` however the method you're making private is `is_of_age?`
- [ ] an easy way to avoid this mistake, especially in a small project is like in the example below.
```
def can_use_services
    is_of_age? || @parent_permission
end

private

def of_age?
    @age >= 18
end
```
This might be easier because all methods below the `private` keyword are automatically made private. and all the methods above the keyword are automatically public so you don't have to define that anymore.